File: okj/rendszerüzemeltetö/fifa vilagranglista/java/Eredmeny.java
public class Eredmeny {
    
    public final String csapat;
    public final int helyezes;
    public final int valtozas;
    public final int pontszam;
    
    public Eredmeny(String line) {
        var split = line.split(";");
        
        csapat = split[0];
        helyezes = Integer.parseInt(split[1]);
        valtozas = Integer.parseInt(split[2]);
        pontszam = Integer.parseInt(split[3]);
    }
}

File: okj/rendszeruzemelteto/fifa_vilagranglista/c#/Eredmeny.cs
public class Eredmeny {
    
    public readonly string csapat;
    public readonly int helyezes;
    public readonly int valtozas;
    public readonly int pontszam;
    
    public Eredmeny(string line) {
        var split = line.Split(';');
        
        csapat = split[0];
        helyezes = int.Parse(split[1]);
        valtozas = int.Parse(split[2]);
        pontszam = int.Parse(split[3]);
    }
}